requirements.txt

```diff
@@ -1,4 +1,4 @@
 colorlog==6.8.2
-pytest-homeassistant-custom-component==0.13.111
+pytest-homeassistant-custom-component==0.13.133
```

Comment:
@danieldotnl , please sign off this change. Previous version did not build the container for me as it could not install mypy-dev==1.10.0a3

Reply:
I already merged that in #372

Reply:
Awesome. Merged with latest.